lua/octo/reviews/layout.lua

```diff
@@ -256,14 +256,14 @@ function Layout:on_enter()
   end
 
   local file = self:get_current_file()
-  if file then
+  if file and config.values.use_local_fs then
```

Comment:
I have some issues with my local file system. but this change is only for local_fs?

Reply:
this change is not needed to fix the bug. because `on_tab_enter` will restore the keymaps. 
The reason I still added this change is because when not using local fs (i.e. octo uses its own buffers), there is no need to delete the keymaps in the first place.